docs/sphinx_doc/source/_templates/package.rst_t

```diff
@@ -1,53 +0,0 @@
-{%- macro automodule(modname, options) -%}
```

Comment:
This file should not be removed.
Remove `docs/sphinx_doc/_templates/package.rst_t` instead